### ISSUE TITLE
This repo and the tutorial on the website are out of sync. Tutorial code will not compile.

### ISSUE THREAD

**Issue by amackillop** [playframework/play-scala-starter-example#123]
The link to download this repo located [here](https://www.playframework.com/documentation/2.7.x/HelloWorldTutorial#Starting-the-project) (as of 2019-03-12) as 'Play Starter Example Project' is out of sync with the rest of the tutorial that follows. This leads to the project being unable to compile if the instructions are followed exactly.

For example, the instructions for making the `hello.scala.html` [here](https://www.playframework.com/documentation/2.7.x/ImplementingHelloWorld#1.-Create-the-Hello-World-page):
```
@main("Hello") {
    <section id="top">
        <div class="wrapper">
            <h1>Hello World</h1>
        </div>
    </section>
}
```
will not work because the `main.scala.html` template expects another explicit parameter which is an `AssetsFinder`. From source file [here](https://github.com/playframework/play-scala-starter-example/blob/2.7.x/app/views/main.scala.html):

`@(title: String, assetsFinder: AssetsFinder)(content: Html)`

The solution is that you need to accept this parameter as an implicit and then pass it to `main` explicitly like so:

```
@()(implicit assetsFinder: AssetsFinder)

@main("Hello", assetsFinder) {
<section id="top">
    <div class="wrapper">
        <h1>Hello World</h1>
    </div>
</section>
}
```

Long story short, I think that either the download link or the tutorial instructions need to be updated in order to resolve this issue.
